src/js/models/panel_model.js

```diff
@@ -158,6 +158,37 @@
             this.save('scalebar', sb);
         },
 
+        show_border: function(color, strokeWidth){
+            var border = {
+                'color': '#'+color,
+                'strokeWidth': strokeWidth,
+                'showBorder':false
```

Comment:
I think this should be `'showBorder':true` and it should be set to `false` with `remove_border` below.